blocks/test/full-content.js

```diff
@@ -95,14 +95,22 @@ describe( 'full post content fixture', () => {
 
 	fileBasenames.forEach( f => {
 		it( f, () => {
+			/**
+			 * Read the input content (HTML + block delimiters).
+			 */
+
 			const content = readFixtureFile( f + '.html' );
 			if ( content === null ) {
 				throw new Error(
 					'Missing fixture file: ' + f + '.html'
 				);
 			}
 
-			const blocksActual = parse( content );
```

Comment:
This declaration should stay below with the rest of the "list of block objects" stuff.  I've added comments to better delineate this code into logical steps.  In a separate PR, we could try adding new `it()` test cases for each of tehse steps.